other-docs/guides/pre-release-modules.md

```diff
@@ -0,0 +1,25 @@
+# Updating Modules to Pre-Release Versions
+
+On occasion when there is a new feature ready for the next major version of Altis that is compatible with the previous version we may create a pre-release version.
+
+Early adopters and those wishing to test or implement the new feature ahead of time can do so by opting in to the new module version.
+
+## Opting in
+
+To opt in to a pre-release of a module you need to require it directly in your root `composer.json`. The command to run is:
+
+```
+composer require altis/<module> <version>@RC --update-with-dependencies
+```
+
+This will override the version constraint in the `altis/altis` meta package. The `@RC` modifier also overrides your project's minimum stability setting.
```

Comment:
Does this override, or does it add `mininum-stability: rc` to the composer.json? If it _does_ do we also need to add `prefer-stable: true`?

Reply:
@joehoyle overrides for that dependency only, will update the wording to be more specific

Reply:
I’m
Not sure I get how this works. Presumably this adds a single line to the composer.json or “2.1.0@RC”, but I don’t get how composer allows installing that if you haven’t set minimum stability? Is it that minimum-stability means “if it say version 3.0, accept 3.0-beta1 if it exists” so it’s still possible to include beta/rc versions without minimum-stability as long as you specify specifically that you want to allow beta versions on that package name? I was under the impression that wasn’t possible